src/main/haskell/kore/src/Kore/Step/AxiomPatterns.hs

```diff
@@ -80,6 +81,7 @@ data AxiomPatternAttributes =
     -- ^ The axiom is an idempotency axiom.
     , trusted :: !Trusted
     -- ^ The claim is trusted
+    , concrete :: !Concrete
```

Comment:
I would prefer that you import this qualified as `Axiom.Concrete` and avoid renaming the existing `Concrete` type. As far as I can tell, this is the only place outside `Kore.Attribute.Axiom.Concrete` that refers to this type.